.github/ISSUE_TEMPLATE/1-bug_report.yaml

```diff
@@ -0,0 +1,204 @@
+name: Bug report
+description: Create a bug report to help us improve
+type: Bug
+labels: [bug]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### Attention!
+
+        Please read [our guide](https://github.com/nvaccess/nvda/blob/master/projectDocs/issues/githubIssueTemplateExplanationAndExamples.md) on how to fill in this template, including how to provide the required files.
+        Issues may be closed if the required information is not present.
+        Please also note that the NVDA project has a [Citizen and Contributor Code of Conduct](https://github.com/nvaccess/nvda/blob/master/CODE_OF_CONDUCT.md).
+        NV Access expects that all contributors and other community members read and abide by the rules set out in this document while participating or contributing to this project.
+        This includes creating or commenting on issues and pull requests.
+
+  - type: textarea
+    id: summary
+    attributes:
+      label: Brief summary
+
+  - type: textarea
+    id: reproSteps
+    attributes:
+      label: Steps to reproduce
+      description: |
+        This section should contain a clear and concise list of the steps you take to demonstrate the problem.
+        While performing the steps required, please make a copy of your [NVDA log to attach to the issue](https://github.com/nvaccess/nvda/wiki/LogFilesAndCrashDumps).
+        It's important to include any files that are required to reproduce an issue.
+        This might be an example file, or a link to a minimal code example using CodePen or JSFiddle.
+
+        *Example:*
```

Comment:
Have you considered putting this in `value` or `placeholder`?

Reply:
I think this would be better in `placeholder`. Do you think we should move all examples there?

Reply:
I've moved these to placeholder